### DISCUSSION TITLE
Support for more MarkSymbolizer styles

### DISCUSSION THREAD

**Discussion by terry-longmacch** [geostyler/geostyler#2534] [General]
Hi Geostylers,

Firstly congratulations on your super project!  We love it, and use it extensively with Pozi here in Australia.

I've just completed some development with our Geostyler forks (geostyler-style, geostyler-sld-parser & geostyler-openlayers-parser) to support the full set of Simple Marker symbols in QGIS and display them in OpenLayers.  I've used SVG to replicate all of the symbols, as the RegularShape styling in OpenLayers is pretty restrictive.  At this stage it is functional for what we need, but requires some tidy up, additional coding in few areas that we currently don't use, and updated unit testing.

I'm happy to do this and provide a pull request, just may need some collaboration to ensure I keep to code styling, testing standards, etc. for the project.

Look forward to hearing from you.

Kind regards,
Terry McDonell

**Comment by KaiVolland** [comment]
Hey @terry-longmacch that sounds great. Looking forward to see your work.
For a more chat like conversation you might want to join our discord: https://discord.gg/yrXHDpcFcB
